packages/amplify-provider-awscloudformation/.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     	"no-param-reassign": "off",
         "no-shadow": "off",
     	"no-return-await" : "off",
-    	"no-continue": "off",
+		"no-continue": "off",
+		"no-plusplus": "off",
```

Comment:
This shouldn't be off. Its inconsistent with other packages style. Style-wise it is recommended to use `i +=1` rather than `i++`.